tests/webapp/api/test_auth.py

```diff
@@ -104,27 +104,6 @@ def test_post_no_auth():
 
 # TC Auth Login and Logout Tests
 
-def test_ldap_login_and_logout(test_user, webapp, monkeypatch):
```

Comment:
once you've created that fixture, you use ``test_ldap_user`` as an argument to the test rather than ``test_user``.